pages/id-card-beta/index.md

```diff
@@ -1,10 +0,0 @@
----
-title: Your VA.gov Account
-layout: page-react.html
-entryname: id-card-beta
----
-<nav class="va-nav-breadcrumbs">
-  <ul class="row va-nav-breadcrumbs-list" role="menubar" aria-label="Primary">
-    <li><a href="/">Home</a></li>
-  </ul>
-</nav>
```

Comment:
Migrated here, https://github.com/department-of-veterans-affairs/vets-website/pull/10660/files#diff-ace45703bb4bd44a29ac5f196c118392R5.